posts/git-cherry-pick/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Cherry-Pick on GitHub"
+author: "Robyn Marowitz"
+date: "2023-09-26"
+description: "Committing a mistake in GitHub can feel so scary, cherry picking is a tool to help you fix your history when something like that happens. This post lays out simple and clear instructions on the cherry picking method."
+categories: [github, open-science]
+image: "cherry_pick.jpeg"
+---
+
+## What is git cherry-pick?
+
+It is the act of picking a commit from one branch and applying it to another branch.  
```

Comment:
A link to the official docs would go really well here! Maybe "Read more in the [official docs](example.com)" at the end?